pkgs/servers/zookeeper/default.nix

```diff
@@ -19,14 +19,14 @@ stdenv.mkDerivation rec {
     mkdir -p $out/bin
     cp -R bin/{zkCli,zkCleanup,zkEnv,zkServer}.sh $out/bin
     patchShebangs $out/bin
+    substituteInPlace $out/bin/zkServer.sh \
+        --replace /bin/echo ${coreutils}/bin/echo \
+        --replace "/usr/bin/env bash" ${bash}/bin/bash
```

Comment:
this one should be fine, and I don't see it in the script
```suggestion
```

Reply:
I just blindly copied that, but you are right, this is not necessary; I have removed it. :-)